CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+* 2.0.0 (@release_date@)
+
+ * dropped support for PHP ^5.2
+ * dropped support for PHP 7.0
+ * dropped support for PHP 7.1
+ * dropped support for PHP 7.2
```

Comment:
Why support a version of PHP not supported by PHP itself? The minimum supported version should be PHP 7.4. Otherwise you're talking about supporting PHP versions that have been EOL for years, and this is going to cause dependency hell as other packages drop support.

Reply:
Huge jumps could lag users behind. So instead of jumping to 7.4 in a sudden, we could move to 5.6, 7.2 and then 7.4, for example.